packages/@lwc/engine-core/src/framework/stylesheet.ts

```diff
@@ -132,6 +133,18 @@ function evaluateStylesheetsContent(
                 // the stylesheet, while internally, we have a replacement for it.
                 stylesheet = getStyleOrSwappedStyle(stylesheet);
             }
+
+            // Check that this stylesheet was generated by our compiler
+            if (!isStylesheetRegistered(stylesheet)) {
+                logError(
+                    `TypeError: Unexpected LWC stylesheet content found for component <${vm.tagName.toLowerCase()}>.`
+                );
```

Comment:
This error should only be logged in dev mode, and should also only be logged once ideally. I'd use the existing `logWarnOnce`, or create a new one called `logErrorOnce`. (In practice, it doesn't matter much whether the text is yellow or red IMO. 😄)

Reply:
Went with `logWarnOnce`. This should now be consistent with how we're reporting and logging the other deprecated features.